### PR TITLE
Homework4

### PR DESCRIPTION
**Pull request by sidelnykov**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,29 +21,37 @@
...
         </dependency>
+
         <!-- https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-java -->
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-java</artifactId>
             <version>3.141.59</version>
         </dependency>
+
+
```
Comment (YegorMaksymchuk) [inline pom.xml:46]: delete extra line

File: pom.xml [hunk 2]
```diff
@@ -21,29 +21,37 @@
...
         <!-- Selenium ChromeDriver -->
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-chrome-driver</artifactId>
             <version>3.14.0</version>
         </dependency>
+
+
+
```
Comment (YegorMaksymchuk) [inline pom.xml:55]: delete two extra line

File: src/main/java/homework4/qaa/MainPage.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+    //Banner in the Header
+    private By headerBanner = By.cssSelector("img[src='http://automationpractice.com/modules/blockbanner/img/sale70.png']");
+
+    //Buttons Contact us and Sign in
+    private By contactUs = By.cssSelector("a[title='Contact Us']");
+    private By signIn = By.cssSelector(".login");
+
+    //Logo
+    private By yourLogo = By.cssSelector("img[src='http://automationpractice.com/img/logo.jpg']");
```
Comment (YegorMaksymchuk) [inline src/main/java/homework4/qaa/MainPage.java:21]: please try next 
By.xpath("//img[contains(@src,'logo.jpg')]");
https://stackoverflow.com/questions/103325/what-is-the-correct-xpath-for-choosing-attributes-that-contain-foo

File: pom.xml (hunk 1 above)
Comment (sidelnykov) [reply pom.xml]: done

File: pom.xml (hunk 2 above)
Comment (sidelnykov) [reply pom.xml]: done

File: src/main/java/homework4/qaa/MainPage.java (hunk 3 above)
Comment (sidelnykov) [reply src/main/java/homework4/qaa/MainPage.java]: done

**Pull request merged**